samtools.wdl

```diff
@@ -25,7 +26,7 @@ task Merge {
           then
             samtools merge ${outputBamPath} ${sep=' ' bamFiles}
           else
-            ln -sf ${bamFiles} ${outputBamPath}
+            ln -sf ${sep=' ' bamFiles} ${outputBamPath}
```

Comment:
I do not agree with this.
Isn't it better to specify this at the workflow level?
When more than one bamfile run this task. When only one bamfile is in the array, this task should not be run in the first place.
Plus, I like WDL logic statements better than bash logic statements. Because WDL logic crashes when something is wrong. Bash inside a command is a bit of a black box. (Sometimes unavoidable, but still)

Reply:
I agree.
@ffinfo can this be changed safely, is this used anywhere else?